docs/admin/admin-ui-setup/templates.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Templates
+summary: This article describes the templates form. This form allows explains what a template is, and where it is used..
+authors:
+    - Jason Novich
+date: 2023-oct-22
+---
+
+A template is a pre-set configuration that is used help quickly configure assets in the system. Using a template allow you to submit a workload based on template which has aggregated all relevant building blocks for your workload.
```

Comment:
This is not true: "A template is a pre-set configuration that is used help quickly configure assets in the system."

Reply:
```suggestion
A template is a pre-set configuration that is used to quickly submit workloads using existing assets.
A template can consist of all the setup a workload needs, thus allowing researchers to submit a workload in a single click.
```

Reply:
```suggestion
A template is a pre-set configuration that is used to quickly configure and submit workloads using existing assets.
A template consists of all the assets a workload needs, allowing researchers to submit a workload in a single click, or make subtle adjustments to differentiate them from each other.
```